protonmail/users.go

```diff
@@ -70,13 +70,28 @@ func (c *Client) GetCurrentUser() (*User, error) {
 		return nil, err
 	}
 
-	var respData struct {
+	var userData struct {
 		resp
 		User *User
 	}
-	if err := c.doJSON(req, &respData); err != nil {
+	if err := c.doJSON(req, &userData); err != nil {
 		return nil, err
 	}
 
-	return respData.User, nil
+	req2, err := c.newRequest(http.MethodGet, "/addresses", nil)
```

Comment:
No need to name this `req2`, you can just overwrite `req` (with `=` instead of `:=`)

Reply:
My mistake. First time using Go :+1: